src/effects/VST3/internal/x11/SocketWindow.h

```diff
@@ -12,9 +12,8 @@
 
 #pragma once
 
-#include <gdk/gdk.h>
-#include <gtk/gtk.h>
-#include <gdk/gdkx.h>
+#include <glib.h>
```

Comment:
The build fixes might be done earlier in the commit sequence to keep all commits buildable.

Reply:
it doesn't build for you?

Reply:
I didn't really try building on Linux.  I may be mistaken.  All commits were building for me on macOS -- but not after the latest rebasing.